Added Personalization brochure (#2137)
ibexa/documentation-developer: docs/personalization/personalization_brochure.md

![Segments](perso_segment_group_or.png)

![Recommendations](recommendations.png)

### REST API

Personalization includes interface as API that you can use to interact with stored data and to send requests and export data from the service.

For more information see documentation for API operations available for Personalization:

- [Tracking API](api_reference/tracking_api.md/)
- [Content API](api_reference/content_api.md/)
- [Recommendation API](api_reference/recommendation_api.md/)
- [User API](api_reference/user_api.md/)

## Benefits

### Web design and content development

Create websites with content targeted at your customers. Deliver your visitors relevant content and build trust in your brand.

### Reduce clutter and improve customer retention

Help users find content of their interest quicker,
use words to appeal to individual customers, as well as segments.
Enrich customer data by integrating Ibexa Personalization with other systems such as ERP, CRMs.
Track user’s scenarios and develop strategies that match your strategies and improve engagement.

> 

Reviewer: unnecessary empty line

> 
> 

Author: ```suggestion

```

> # Personalization

Reviewer 2: ```suggestion
# Personalization product guide
```

## What is Personalization

Personalization engine is a cloud-based service that tracks and analyzes customer behaviors and delivers optimized, user-tailored content, and information.

> The engine uses artificial intelligence and machine learning modelling to learn and create behavioral patterns and generate recommendations ready to be presented on one or more websites hosted by the Ibexa DXP instance.

Reviewer 2: ```suggestion
The engine uses artificial intelligence and machine learning modeling to learn and create behavioral patterns and generate recommendations ready to be presented on one or more websites hosted by the [[= product_name =]] instance.
```

## Availability

> Personalization service is available in all supported Ibexa DXP versions and editions.

Reviewer 2: ```suggestion
Personalization service is available in all supported [[= product_name =]] versions and editions.
```

Pricing is based on the number of Personalization API events.

> To enable recommendation engine, request access to the service by [creating an account](https://doc.ibexa.co/projects/userguide/en/master/personalization/enable_personalization/#request-access-to-the-server).

Reviewer 2: ```suggestion
To enable recommendation engine, request access to the service by [creating an account]([[= user_doc =]]/personalization/enable_personalization/#request-access-to-the-server).
```

After setting up an account and receiving credentials, pass them to your administrator who can configure Personalization service in your [[= product_name =]] instance.

Now, you can start collecting data and boost your business.

> ## How does Personalization work?

Reviewer 2: ```suggestion
## How does Personalization work
```
you don't have question marked in the first header

![How it works](how_perso_works.png)

Personalization with tracking scripts, monitors  individual sessions from user’s behavior and interest data through events (clicks, viewed articles, pages, purchases) and demographic data (location, industry, occupation).

Next, collects data and computes models thanks to complex algorithms. 

Personalization is SiteAcces-aware, it means it can serve on multiple sites with different customer IDs and deliver recommendations for each site (for example, if you host multi language site).

Use Twig extension to add a tracking script and implement it into your site configuration.

## Capabilities

### Models

Models are statistics-based and perform calculations based on data from content, users and events.

They work in the background and update on regular time intervals to ensure best and accurate recommendations. Personalization service includes a few predefined models according to your needs and agreements between your organization and Ibexa.

If you need more customized models, contact customer support.

> Includes the following model types:

Reviewer 2: ```suggestion
Personalization service includes the following model types:
```

- editorial models - returns items, products from the manually prepared list

- B2B models - returns items clicked or bought by a segment group of a company

![Structure](categories.png)

### Scenarios

Scenarios are sequences of events. You can use scenarios to determine how and when render particular content to your customers. Each scenario can include also subscenarios used for granular personalized rendering of content.

### Segments

Use segments to get content targeted at particular user groups. Segments calculate models based on the segment attribute factor. Assign users to different recommendation groups and create advanced logic with operators to get best segmentation for your audience.

![Segments](perso_segment_group_or.png)

![Recommendations](recommendations.png)

### REST API

Personalization includes interface as API that you can use to interact with stored data and to send requests and export data from the service.

For more information see documentation for API operations available for Personalization:

> [Tracking API](api_reference/tracking_api.md/)
> 
> [Content API](api_reference/content_api.md/)
> 
> [Recommendation API](api_reference/recommendation_api.md/)
> 
> [User API](api_reference/user_api.md/)

Reviewer 2: ```suggestion
- [Tracking API](api_reference/tracking_api.md/)
- [Content API](api_reference/content_api.md/)
- [Recommendation API](api_reference/recommendation_api.md/)
- [User API](api_reference/user_api.md/)
```

## Benefits

### Web design and content development

Create websites with content targeted at your customers. Deliver your visitors relevant content and build trust in your brand.

### Reduce clutter and improve customer retention

> Help users find content of their interest quicker
> 
> use words to appeal to individual customers, as well as segments
> 
> Enrich customer data by integrating Ibexa Personalization with other systems such as ERP, CRMs.
> 
> Track user’s scenarios and develop strategies that match your strategies and improve engagement

Reviewer 2: ```suggestion
Help users find content of their interest quicker,
use words to appeal to individual customers, as well as segments.
Enrich customer data by integrating Ibexa Personalization with other systems such as ERP, CRMs.
Track user’s scenarios and develop strategies that match your strategies and improve engagement.
```

### Satisfy customer expectations

> Personalization engine shows customers products relevant to their interests, builds a feeling of connection and understanding

Reviewer 2: ```suggestion
Personalization engine shows customers products relevant to their interests, builds a feeling of connection and understanding.
```

### Improve customer experience

Stand out from other companies by provided well-suited content and make a space for your customers where they feel known and valued and treated exceptional. Build trust and a bond with visitors, who are more likely to come back.

### Increase average order value 

Apply collaborative models with predictive analysis and find out what motivates users to put extra items into their carts. Start building predictions of their behaviors and suggest items, products your visitors are willing to buy.

![Dashboard](perso_dashboard_revenue.png)

### Measure your return and boost conversion rates 

Connect Personalization engine with your eCommerce shop, focus on the revenue generated by recommendations. Track what recommendations are shown to your visitors and analyze conversion rates. Effectively evaluate quality of recommendations and compare with your KPIs.

## Use cases

> [eCommerce]([[= user_doc =]]/personalization/use_cases/#ecommerce.md)
> 
> [Content publishing]([[= user_doc =]]/personalization/use_cases/#content-publishing.md)
> 
> [Multiple website hosting]([[= user_doc =]]/personalization/use_cases/#multiple-website-hosting.md)

Reviewer 2: ```suggestion
- [eCommerce]([[= user_doc =]]/personalization/use_cases/#ecommerce.md)
- [Content publishing]([[= user_doc =]]/personalization/use_cases/#content-publishing.md)
- [Multiple website hosting]([[= user_doc =]]/personalization/use_cases/#multiple-website-hosting.md)
```
